src/gwc/src/main/java/org/geoserver/gwc/controller/DispatcherController.java

```diff
@@ -0,0 +1,28 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
This is a new file, so the header here looks incorrect. Is this due to a rename not tracked by github?